common/djangoapps/philu_commands/management/commands/autoscore_ondemand_course_ora.py

```diff
@@ -58,52 +54,17 @@ def handle(self, *args, **options):
-                                                log.info('Multiple Anonymous Ids for User: {user}'.format(user=user))
-                                                continue
-
-                                            response_submission = Submission.objects.filter(
-                                                student_item__student_id=anonymous_user.anonymous_user_id,
-                                                student_item__item_id=block).first()
-                                            if not response_submission:
-                                                continue
-                                            log.info('Response Created at: {created_date}'.format(
-                                                created_date=response_submission.created_at.date()
-                                            ))
-
-                                            response_submission_delta = today - response_submission.created_at.date()
-
-                                            # check if this chapter is 2 weeks older or not.
-                                            module_access_days = delta_days.days - (index_chapter * 7)
-
-                                            if (module_access_days >= DAYS_TO_WAIT_AUTO_ASSESSMENT and
-                                                    response_submission_delta.days >= DAYS_TO_WAIT_AUTO_ASSESSMENT):
-                                                try:
-                                                    # Status[0] is the status of assessment that are in waiting mode
-                                                    AssessmentWorkflow.objects.get(
-                                                        status=AssessmentWorkflow.STATUSES[0],
-                                                        course_id=course.id,
-                                                        item_id=block,
-                                                        submission_uuid=response_submission.uuid
-                                                    )
-                                                    student = {
-                                                        'id': user.id,
-                                                        'username': user.username,
-                                                        'email': user.email,
-                                                        'anonymous_user_id': anonymous_user.anonymous_user_id
-                                                    }
-                                                    autoscore_ora(course.id, unicode(block), student)
-                                                except AssessmentWorkflow.DoesNotExist:
-                                                    continue
+                                        if block.block_type == ORA_BLOCK_TYPE:
+
+                                            if can_auto_score_ora(user, course, block, delta_days,
```

Comment:
How about combining both of these conditions in a single if?

Reply:
Done
Please re-review